docs/data-operations-manual/Reference/Performance-db-tables-construction.md

```diff
@@ -0,0 +1,186 @@
+## Performance Database Table Construction Documentation
+
+This is documentation that explains the construction of the tables in the [performance database](https://datasette.planning.data.gov.uk/performance/), including details on base tables, sources of columns.
+
+The Performance Database is designed to store and analyze performance-related metrics extracted from the source database [digital_land](https://datasette.planning.data.gov.uk/digital-land). The tables of this DB are:
+
+>1. `endpoint_dataset_issue_type_summary`
+>2. `endpoint_dataset_resource_summary`
+>3. `endpoint_dataset_summary`
+>4. `provision_summary`
+>5. `reporting_historic_endpoints`
+>6. `reporting_latest_endpoints`
+
+### 1. Table: [endpoint_dataset_issue_type_summary](https://datasette.planning.data.gov.uk/performance/endpoint_dataset_issue_type_summary)
+
+**Purpose**: To summarize issues associated with each dataset and endpoint.
```

Comment:
Maybe include something about the organisation. And say dataset and its associated endpoints